bson/src/test/resources/bson/binary.json

```diff
@@ -39,6 +39,12 @@
             "canonical_bson": "1D000000057800100000000473FFD26444B34C6990E8E7D1DFC035D400",
             "canonical_extjson": "{\"x\" : { \"$binary\" : {\"base64\" : \"c//SZESzTGmQ6OfR38A11A==\", \"subType\" : \"04\"}}}"
         },
+        {
+            "description": "subtype 0x04 UUID",
+            "canonical_bson": "1D000000057800100000000473FFD26444B34C6990E8E7D1DFC035D400",
+            "canonical_extjson": "{\"x\" : { \"$binary\" : {\"base64\" : \"c//SZESzTGmQ6OfR38A11A==\", \"subType\" : \"04\"}}}",
+            "degenerate_extjson": "{\"x\" : { \"$uuid\" : \"73ffd264-44b3-4c69-90e8-e7d1dfc035d4\"}}"
```

Comment:
1. How do you generate that BSON string?
2. Can you explain what the degenerate_extjson represents in these tests? Is it just an alternative to the canonical way of doing it?

Reply:
1. I didn't do this myself as it's from the specs repo, but you can generate this by first converting to BSON using any BSON encoder, then hex-encoding the bytes.  I can show you how to do it with the Java BSON library if you'd like
2. See https://github.com/mongodb/specifications/blob/master/source/bson-corpus/bson-corpus.rst for a description of the test file format